spec/controllers/selects_collections_spec.rb

```diff
@@ -7,28 +7,27 @@ class SelectsCollectionsController < ApplicationController
 end
 
 describe SelectsCollectionsController, :type => :controller do
+  let(:user) { FactoryGirl.find_or_create(:jill) }
+  let(:dit) { FactoryGirl.find_or_create(:dit) }
   describe "Select Communities" do
-    before :all do
+    before :each do
```

Comment:
why the community/collection need to be set up for each test? Can we reuse them for all the tests in this spec?